python/pyfeelpp-mor/tests/test_nirb.py

```diff
@@ -0,0 +1,98 @@
+import sys
+import os
+import pytest
+
+
+import feelpp
+from feelpp.mor.nirb.nirb import *
+
+# desc : ((toolboxtype, 'model_directory', cfg, json, geo, H, h, dimension, doRectification, doGreedy), 'name-of-the-test')
+casesNirb = [
+        #  (('testcase/nirb/lid-driven-cavity/', 'cfd2d.cfg', 'cfd2d.json', False), 'lid-driven-cavity w/o rect.'),
+        #  (('testcase/nirb/lid-driven-cavity/', 'cfd2d.cfg', 'cfd2d.json', True) , 'lid-driven-cavity rect'),
+         (('testcase/nirb/square', 'square.cfg', 'square.json', False, False), 'square2d w/o rect wogreedy'),
+         (('testcase/nirb/square', 'square.cfg', 'square.json', True, False) , 'square2d rect wogreedy'),
+         (('testcase/nirb/square', 'square.cfg', 'square.json', True, True) , 'square2d rect egreedy'),
```

Comment:
the test with Greedy selection dosn't get an error less than 8.10-2. I remove it now by waiting its new version.